servers/lib/src/files/local/local-files.service.ts

```diff
@@ -9,6 +9,9 @@ import { IFilesService } from '../interfaces/files.service.interface.js';
 export default class LocalFilesService implements IFilesService {
   // eslint-disable-next-line no-useless-constructor, no-empty-function
   constructor(private configService: ConfigService) {}
+  getFileMode(): string {
+    return 'local';
+  }
 
   async listDirectory(path: string): Promise<Project> {
     const dataPath = this.configService.get('LOCAL_PATH');
```

Comment:
The `dataPath` variable does not change during the execution of the microservice. Can this be moved to constructor? Are there any implications for testing if it is moved there? Same is the case with `git-files.service.ts`.